examples/folder-level-config/main.tf

```diff
@@ -0,0 +1,47 @@
+provider "google" {}
```

Comment:
I would suggest we remove the "folder level" example, because this is not real folder level integration. This is still project level integration. Customer should be able to figure it out by themselves to use the `google_cloud_asset_search_all_resources`

Reply:
As discussed, I updated the name from folder-level-config to project-level-config-by-folder to indicate that it is still by project level. Once we support folder level, will add new examples.